zcash_client_backend/CHANGELOG.md

```diff
@@ -65,6 +82,8 @@ and this library adheres to Rust's notion of
 ### Removed
 - `zcash_client_backend::PoolType::is_receiver`: use
   `zcash_keys::Address::has_receiver` instead.
```

Comment:
```suggestion
- `zcash_client_backend::PoolType::is_receiver` (use
  the `zcash_keys::Address::has_receiver` method instead).
```
for consistency with Str4d's suggestion.

Reply:
This can be handled in PR cleanup, as it's not affecting changes of this PR.